examples/all-clusters-app/all-clusters-common/gen/af-structs.h

```diff
@@ -190,6 +190,24 @@ typedef struct _GroupInformationRecord
     uint8_t groupType;
 } EmberAfGroupInformationRecord;
 
+// Struct for GroupKeys
+typedef struct _GroupKeys
+{
+    uint16_t VendorId;
+    uint16_t GroupKeyIndex;
+    uint16_t GroupKeyRoot;
+    uint64_t GroupKeyEpochStartTime;
+    uint8_t GroupKeySecurityPolicy;
+} EmberAfGroupKeys;
```

Comment:
This is likely more broad than just this PR, but is there a plan for moving away from "Ember" naming similar to how "Weave" references were removed from that input technology?

Reply:
There is.  It's generally been blocked on the fact that not all the generated code was easy to regenerate and it includes the "ember" naming in it.  The plan of record is that once we are fully on ZAP generation we can start renaming things.  I think we might be there, in which case we can just start doing it.

Reply:
The plan is to replace "ember" with "chip" in ZAP generator.